tsdb/engine/tsm1/engine.go

```diff
@@ -377,11 +371,11 @@ type devTx struct {
 func (t *devTx) Cursor(series string, fields []string, dec *tsdb.FieldCodec, ascending bool) tsdb.Cursor {
 	if len(fields) == 1 {
 		return &devCursor{
-			tsm:       t.engine,
-			series:    series,
-			fields:    fields,
-			cache:     t.engine.Cache.Values(SeriesFieldKey(series, fields[0])),
-			ascending: ascending,
+			series:       series,
+			fields:       fields,
+			cache:        t.engine.Cache.Values(SeriesFieldKey(series, fields[0])),
+			tsmKeyCursor: t.engine.KeyCursor(SeriesFieldKey(series, fields[0])),
```

Comment:
Does cursor by this point only have one field or are we going to have to fix this for queries that hit multiple field?